packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -846,75 +805,80 @@
 			]
 		},
 		"50": {
-			"label": "Active power reports",
-			"description": "Pwr level change resulting in power report sent",
+			"label": "Active power report",
+			"description": "The power level change as a percentage from the previous report that will result in a new power report being sent",
 			"valueSize": 1,
+			"unit": "%",
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 10
+			"defaultValue": 10,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		"52": {
 			"label": "Periodic active power and energy reports",
-			"description": "Time period between consecutive reports",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 32767,
-			"defaultValue": 3600
+			"defaultValue": 3600,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
 		},
 		"53": {
-			"label": "Energy reports",
-			"description": "Energy level change resulting in energy report sent",
+			"label": "Energy report",
+			"description": "The energy level change resulting in energy report sent. Each step is 0.01 kWh, e.g. 10 = 0.1 kWh",
```

Likewise:
```suggestion
			"label": "Energy Report: Change Threshold",
```
and the unit should be put in the `"unit"` property.

---

Here I thought about if I should use `"unit" : "0.01 kWh",` or `"unit" : "10 Wh",` any guidelines for that?

---

No real guidelines, but we normally use the `kWh` variant.